NitroxClient/GameLogic/PlayerManager.cs

```diff
@@ -75,12 +77,22 @@ public void RemovePlayer(ushort playerId)
             {
                 opPlayer.Get().Destroy();
                 playersById.Remove(playerId);
+                DiscordController.Main.UpdateDRPDiving(GetPlayerCount() + 1);
```

Comment:
Shouldn't this be `GetPlayerCount() -1`?

Reply:
No, the "+1" is for the player himself because "GetPlayerCount()" just return the "multiplayer-player".